R/initialize_project.R

```diff
@@ -63,28 +80,27 @@ initialize_project <- function(folders = c('scripts', 'original_data',
 #' can be helpful when working with existing projects. See `initialize_project()`
 #' for details on adding template files for new organizations.
 #'
-#' @param to.path Destination file path for template files. Typically, root of Rproject directory.
+#' @param to.path Character string. Destination file path for template files. Typically, root of Rproject directory.
+#' @param overwrite Boolean. Overwrite _quarto.yml and style.css files if they already exist? Defaults to FALSE.
 #' @inheritParams initialize_project
 #'
 #' @return Nothing.
 #' @export
 #'
-fetch_quarto_templates = function(to.path, organization = "WDFW"){
+fetch_quarto_templates = function(to.path, organization = "WDFW", overwrite = FALSE){
```

Comment:
maybe something like

```r
fetch_quarto_templates = function(to.path, organization = c("WDFW"), overwrite = FALSE){  
organization <- rlang::arg_match(organization)
}
```

Reply:
depends on where you want to put it - up higher in `initialize_project()` and it'll stop the code before anything happens, down in `fetch_quarto_templates()` it'll abort after the directories have been copied.

Reply:
Awesome, thanks! I want fetch_quarto_templates() to work on its own, but I will add the suggested approach to both of them.